CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+Thanks for your interest in contributing to Cookie Though! Please take a moment to review this document **before submitting a pull request**.
+
+## Pull requests
+
+**Please ask first before starting work on any significant new features.**
+
+It's never a fun experience to have your pull request declined after investing a lot of time and effort into a new feature. To avoid this from happening, we request that contributors create [an issue](https://github.com/inthepocket/cookie-though/issues) to first discuss any significant new features. This includes things like adding new utilities, creating new style rules, etc.
+
+## Coding standards
+
+Our code formatting rules are defined in [.eslintrc.js](https://github.com/inthepocket/cookie-though/blob/master/.eslintrc.js). You can check your code against these standards by running:
+
+```console
```

Comment:
Note that the correct language for console entries is console, not `sh` which means the sh shell specifically or sh scripts.